packages/web/src/pages/settings-page/components/desktop/ManagerMode/ConfirmAccountManagerPage.tsx

```diff
@@ -0,0 +1,124 @@
+import { useCallback, useState } from 'react'
+
+import { accountSelectors } from '@audius/common/store'
+import { encodeHashId } from '@audius/common/utils'
+import {
+  Button,
+  Flex,
+  Hint,
+  IconCaretLeft,
+  IconError,
+  Text,
+  TextLink,
+  Box
+} from '@audius/harmony'
+
+import ArtistChip from 'components/artist/ArtistChip'
+import { audiusSdk } from 'services/audius-sdk'
+import { reportToSentry } from 'store/errors/reportToSentry'
+import { useSelector } from 'utils/reducer'
+
+import { sharedMessages } from './sharedMessages'
+import {
+  AccountsManagingYouPages,
+  ConfirmAccountManagerPageProps
+} from './types'
+
+const { getUserId } = accountSelectors
+
+const messages = {
+  description:
+    'Are you sure you want to authorize the following user to manage your Audius account?',
+  confirm: 'Confirm',
+  back: 'Back',
+  errorNoUserId: 'Something went wrong - please refresh and try again.',
```

Comment:
Can't give a definitive answer for that, it'd be something very unexpected lol. Put this here so we can show something in case the `if (!userId)` ever happens below (since the `getUserId` selector return type is `User | null`).